src/openrct2/ride/Vehicle.h

```diff
@@ -76,8 +77,9 @@ struct rct_ride_entry_vehicle {
     uint8 effect_visual;
     uint8 draw_order;
     uint8 num_vertical_frames_override; // 0x60 , 0x7A, A custom number that can be used rather than letting RCT2 determine it. Needs the VEHICLE_ENTRY_FLAG_OVERRIDE_NUM_VERTICAL_FRAMES flag to be set.
-    sint8* peep_loading_positions;  // 0x61 , 0x7B
-    uint16 peep_loading_positions_count;
+    uint8 pad_61[7];                // 0x61 , 0x7B
```

Comment:
There is no need to pad this we already own the format and have made changes to it.

Reply:
The vector needs to be aligned to 8. We still need pack 1 for testpaint.

Reply:
Maybe it would make sense to limit the packing to testpaint-only then?

Reply:
`#ifdef __TESTPAINT__`

Reply:
@IntelOrca it looks like CIs are not able to ensure the 8-byte alignment for 32-bit code, but I guess we only need 4-byte alignment in those cases anyway?

Reply:
Probably.